src/3d/shaders/instanced.frag

```diff
@@ -6,6 +6,7 @@ uniform vec3 ka;                            // Ambient reflectivity
 uniform vec3 kd;                            // Diffuse reflectivity
 uniform vec3 ks;                            // Specular reflectivity
 uniform float shininess;                    // Specular shininess factor
+uniform float opacity;
```

Comment:
Missing a comment here, but is `// opacity` really useful? :)

Reply:
Yes, not that much useful, but I have added it anyway.